src/User32.Desktop/User32.cs

```diff
@@ -621,10 +674,478 @@ public static partial class User32
+        /// <param name="lParam">An application-defined value to be passed to the callback function.</param>
+        /// <returns>
+        /// If the function is unable to perform the enumeration, the return value is zero. Call GetLastError to get extended error information.
+        /// If the function succeeds, it returns the nonzero value returned by the callback function that was pointed to by <paramref name="lpEnumFunc"/>.
+        /// If the callback function fails, the return value is zero. The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </returns>
+        /// <remarks>
+        /// The EnumWindowStations function enumerates only those window stations for which the calling process has the <see cref="WINDOW_STATION_ACCESS_MASK.SpecificRight.WINSTA_ENUMERATE"/> access right.
+        ///  EnumWindowStations repeatedly invokes the <paramref name="lpEnumFunc"/> callback function until the last window station is enumerated or the callback function returns FALSE.
+        /// </remarks>
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool EnumWindowStations(EnumWindowStationDelegate lpEnumFunc, IntPtr lParam);
+
+        /// <summary>
+        /// Enumerates all window stations in the current session. The function passes the name of each window station, in turn, to an application-defined callback function.
+        /// </summary>
+        /// <param name="lpEnumFunc">An application-defined <see cref="EnumWindowStationDelegate"/> callback function.</param>
+        /// <param name="lParam">An application-defined value to be passed to the callback function.</param>
+        /// <returns>
+        /// If the function is unable to perform the enumeration, the return value is zero. Call GetLastError to get extended error information.
+        /// If the function succeeds, it returns the nonzero value returned by the callback function that was pointed to by <paramref name="lpEnumFunc"/>.
+        /// If the callback function fails, the return value is zero. The callback function can call SetLastError to set an error code for the caller to retrieve by calling GetLastError.
+        /// </returns>
+        /// <remarks>
+        /// The EnumWindowStations function enumerates only those window stations for which the calling process has the <see cref="WINDOW_STATION_ACCESS_MASK.SpecificRight.WINSTA_ENUMERATE"/> access right.
+        ///  EnumWindowStations repeatedly invokes the <paramref name="lpEnumFunc"/> callback function until the last window station is enumerated or the callback function returns FALSE.
+        /// </remarks>
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool EnumWindowStations(IntPtr lpEnumFunc, IntPtr lParam);
```

Comment:
What are these specialized functions you refer to? Are you saying that you have methods that match the signature but (for example) consider the last parameter to be an enum instead of an `int`, making them unable to be typed in .NET as the proper callback delegate, such that getting the function pointer is easier?
If so, I see the importance of offering the `IntPtr` overload. And I see the value of the Friendly-style code generator to help. Can you file an issue for enhancing the generator to offer `IntPtr` overloads for all such methods that accept delegates? And then, if you have time, please add overloads of these methods so that at least there are delegate-typed methods for all these. That way, when the code generator is ready, we can just remove the manually coded `IntPtr` overloads.